### ISSUE TITLE
only signed in can comment

### ISSUE THREAD

**Issue by ogurec-ogurec** [danirus/django-comments-xtd#208]
Faced a small problem, is it possible to somehow make it so that only users can comment, but at the same time the "Reply" button and the "Like / Dislike" buttons were displayed, and when they clicked on which, they were also sent to the authorization page ? (according to this logic, the "Flag" button is displayed).
 
Currently, only the flag is displayed, and changes to the settings do not affect the rendering of the form. (like / dislike, reply buttons are not shown, even when the show_feedback setting is enabled)

Maybe someone came across?

My settings:
```
'allow_flagging': True,
'allow_feedback': True,
'show_feedback': True,
'who_can_post': 'users'
```

**Comment by danirus** [comment]
Hi @ogurec-ogurec, sorry for the late reply.
The interface displayed by the JavaScript plugin should be the same as the backend, when this feature is enabled.
I consider it a bug, I will fix it and release it in the v2.7.2.
Thanks for the report!

**Comment by danirus** [comment]
The issue has been fixed and will be release with v2.7.2.

**Comment by ogurec-ogurec** [comment]
thanks!
